app/views/standard_layout.scala.html

```diff
@@ -109,7 +109,7 @@
                             <li><a href="@uk.gov.hmrc.cataloguefrontend.shuttering.routes.ShutterEventsController.shutterEvents">Shutter Events</a></li>
                             <li role="separator" class="divider"></li>
                             <li><a href="@uk.gov.hmrc.cataloguefrontend.leakdetection.routes.LeakDetectionController.ruleSummaries">Leak Detection - Rules</a></li>
-                            <li><a href="@uk.gov.hmrc.cataloguefrontend.leakdetection.routes.LeakDetectionController.repoSummaries">Leak Detection - Repositories</a></li>
+                            <li><a href="@uk.gov.hmrc.cataloguefrontend.leakdetection.routes.LeakDetectionController.repoSummaries?includeViolations=true">Leak Detection - Repositories</a></li>
```

Comment:
for type safety, can we not update the route definition to take the form params? ([e.g.](https://github.com/hmrc/catalogue-frontend/blob/main/conf/app.routes#L71))